### PR TITLE
Feature/create player model

### PR DESCRIPTION
**Pull request by eSmelser**
@bobbyesh if you wouldn't mind reviewing this to see what you think about the implementation of the data model. I tried to make everything mostly speak for itself, and I added session and sub to the data model I had initially intended. I believe these help clean up the model so that it works in the way I'd intended. I really would like to do some db verification before merging this in, so let me know if you know of a good way to do that with Django if you have time! 

Thanks again.

Let me know if you have any questions.

### CODE REVIEW

**Comment by eSmelser** [comment]
@bobbyesh It doesn't let me add you as a reviewer, but that was my intention. I'll look into it.

File: SnookR/main/models.py [hunk 1]
```diff
@@ -1,3 +1,85 @@
...
+'''
+class Division(models.Model):
+	name         = models.CharField(max_length=200)
+	division_rep = models.ForeignKey(Player, related_name='division_representative')
+	teams        = models.ManyToManyField(Team, blank=True, related_name="divisions_teams")
+	subs         = models.ManyToManyField(Sub, blank=True, related_name="divisions_subs")
+
+	def __str__(self):
+		return self.name
+
```
Comment (eSmelser) [inline SnookR/main/models.py:69]: Not sure about how this division will interact with the sublist model. @bobbyesh any thoughts?

File: SnookR/main/models.py [hunk 2]
```diff
@@ -1,3 +1,85 @@
...
+Player -> Division : 0 .. *
+Player -> Sub      : 
+'''
+class Player(models.Model):
+	username     = models.ForeignKey(User, related_name="players_username")
+	first_name   = models.CharField(max_length=50)
+	last_name    = models.CharField(max_length=50)
+	phone_number = models.IntegerField(blank=True)
+	email        = models.CharField(max_length=200, blank=True)
+
```
Comment (eSmelser) [inline SnookR/main/models.py]: Refactor to use User's `name` and `username`  related fileds.

**Comment by bobbyesh** [comment]
My bad, JUST saw this.  Looking over now...

**Pull request merged**